test/js/geo/transform.test.js

```diff
@@ -141,7 +141,7 @@ test('vertex buffer', function(t) {
     t.test('is initialized', function(t) {
         var buf = new VertexBuffer();
         t.deepEqual(buf.index, 0);
-        t.deepEqual(buf.length, 32768);
+        t.deepEqual(buf.capacity, 8192);
```

Comment:
`capacity` isn't the same as `length`. `capacity` has to do with how much memory we have allocated for the buffer. `length` has to do with how many elements are in the buffer. I would hope that the `length` wouldn't actually change in this test. Do you know if `length` is still wrong here? By how much?